### PR TITLE
Add types for tuya-panel-kit

### PR DESCRIPTION
**Pull request by youngjunning**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## tuya-panel-kit ([<kbd>unpkg</kbd>](https://unpkg.com/browse/tuya-panel-kit@latest/))
was missing the following properties:
1. OfflineView
1. FullView
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cb608cc0fd981a79c41cb5e62bdad8526c8341d7
</p>

**Comment by youngjuning** [comment]
anyone here？

File: types/tuya-panel-kit/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,2320 @@
...
+    ListRenderItem,
+} from 'react-native';
+import { ParamListBase, RouteProp } from './@react-navigation/native';
+import { StackNavigationOptions } from './@react-navigation/stack';
+import { GlobalTheme, StopsProps, LinearGradientBackground, RadialGradientBackground, BackgroundProps } from './theme';
+
+export { GlobalTheme } from './theme';
+
+// tslint:disable-next-line  no-declare-current-package
+declare module 'tuya-panel-kit' {
```
Comment (sandersn) [inline types/tuya-panel-kit/index.d.ts:38]: Why have an export, plus a nested `declare module`? Does the nested declare module allow use of its contents without importing them?

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/tuya-panel-kit/index.d.ts (hunk 1 above)
Comment (youngjuning) [reply types/tuya-panel-kit/index.d.ts]: I have removed it.:)

**Review by sandersn — APPROVED** [review]

**Pull request merged**